backend/src/test/java/com/pickpick/acceptance/BookmarkAcceptanceTest.java

```diff
@@ -23,4 +29,45 @@ public class BookmarkAcceptanceTest extends AcceptanceTest {
         // then
         상태코드_확인(response, HttpStatus.CREATED);
     }
+
+    @Test
+    void 멤버_ID_2번으로_북마크_조회() {
+        // given
+        Map<String, Object> request = Map.of("bookmarkId", "");
+        List<Long> expectedIds = List.of(1L);
+        boolean expectedIsLast = true;
+
+        // when
+        ExtractableResponse<Response> response = getWithAuth(API_BOOKMARK, 2L, request);
+
+        // then
+        상태코드_확인(response, HttpStatus.OK);
+
+        BookmarkResponses bookmarkResponses = response.jsonPath().getObject("", BookmarkResponses.class);
+        assertThat(bookmarkResponses.isLast()).isEqualTo(expectedIsLast);
+        assertThat(convertToIds(bookmarkResponses)).containsExactlyElementsOf(expectedIds);
+    }
+
+    @Test
+    void 멤버_ID_1번이고_북마크_ID가_23번일_때_북마크_목록_조회() {
+        // given
+        Map<String, Object> request = Map.of("bookmarkId", "23");
+        List<Long> expectedIds = List.of(22L, 21L, 20L, 19L, 18L, 17L, 16L, 15L, 14L, 13L, 12L, 11L, 10L, 9L, 8L, 7L,
+                6L, 5L, 4L, 3L);
+        boolean expectedIsLast = false;
+
+        // when
+        ExtractableResponse<Response> response = getWithAuth(API_BOOKMARK, 1L, request);
+
+        // then
+        상태코드_확인(response, HttpStatus.OK);
+
+        BookmarkResponses bookmarkResponses = response.jsonPath().getObject("", BookmarkResponses.class);
+        assertThat(bookmarkResponses.isLast()).isEqualTo(expectedIsLast);
+        assertThat(convertToIds(bookmarkResponses)).containsExactlyElementsOf(expectedIds);
+    }
+
+    private List<Long> convertToIds(final BookmarkResponses response) {
+        return response.getBookmarks().stream().map(BookmarkResponse::getId).collect(Collectors.toList());
+    }
```

Comment:
```suggestion
    private List<Long> convertToIds(final BookmarkResponses response) {
        return response.getBookmarks()
        .stream()
        .map(BookmarkResponse::getId)
        .collect(Collectors.toList());
    }
```

tc이긴 하지만 개행을 추가하는건 어떨까요?